live-examples/js-examples/promise/promise-constructor.html

```diff
@@ -1,9 +1,23 @@
 <pre>
 <code id="static-js">var promise1 = new Promise(function(resolve, reject) {
-  setTimeout(resolve, 100, 'foo');
```

Comment:
Using `setTimeout` like this, with a 3rd argument ('foo') -  is IMHO rather rare.